code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -309,7 +309,7 @@
 	else if(istype(AM, /obj/structure/closet))
 		return FALSE
 	else if(isobj(AM))
-		if((!allow_dense && AM.density) || AM.anchored || AM.has_buckled_mobs())
+		if((!allow_dense && AM.density) || AM.anchored || AM.has_buckled_mobs() || ismecha(AM))
```

Comment:
Won't this break supplypods?

Reply:
That's definitely something I overlooked, so I'm not sure. I'll go back and do some testing and revising if necessary.

Reply:
> Won't this break supplypods?

Did some testing, sending mechs in supply pods is still completely functional.

Reply:
what about the reverse supply pods, which have an option to grab mechs that are on the ground?

Reply:
> what about the reverse supply pods, which have an option to grab mechs that are on the ground?

Did another test, and thankfully yes they still work if the option to pick up mechs is enabled.

Reply:
okay thank you